splunk_add_on_ucc_framework/install_python_libraries.py

```diff
@@ -109,15 +107,21 @@ def _pip_is_lib_installed(
         # Disable writing of .pyc files (__pycache__)
         my_env["PYTHONDONTWRITEBYTECODE"] = "1"
 
-        if allow_higher_version:
-            result = _subprocess_run(command=cmd, env=my_env)
+        result = _subprocess_run(command=cmd, env=my_env)
+        if result.returncode != 0:
+            cmd_windows = cmd.replace("grep", "findstr")
```

Comment:
Do we need `grep` or `findstr` in general? Can we do this parsing in Python itself and not outsource to a OS-dependent call?

I do not want to block this PR because of this but if that's possible - it would be nice to follow up with a refactor after the release.

Reply:
We can but this was the simplest one. My alternative idea was to do some sys.path insertion for `output/addon/lib`, next `import splunktaucclib` and then simply call `__version__` . We can also parse whole pip show output using python. Probably I won't be able to finish this today but I believe I can refactor this tomorrow.

Reply:
Neither grep nor findstr is needed for just checking if a string is in command's output. We can do something like this:

```
result = _subprocess_run(command=lib_installed_cmd, env=my_env)

if result.returncode != 0 or f"Version: {version}" not in result.stdout.decode():
    return False
```